starfish/pipeline/features/spots/detector/gaussian.py

```diff
@@ -95,7 +96,9 @@ def _measure_spot_intensities(
         n_ch = stack.shape[Indices.CH]
         n_hyb = stack.shape[Indices.HYB]
         spot_attribute_index = dataframe_to_multiindex(spot_attributes)
-        intensity_table = IntensityTable.empty_intensity_table(spot_attribute_index, n_ch, n_hyb)
+        image_shape: Tuple[int, int, int] = stack.raw_shape[2:]
```

Comment:
why do you want to call raw_shape?  if you have specific indices you care about (which you do), why not just call shape on the specific indices in the order you want it in?

specifically, if we rearrange the order of the axes in ImageStack, would this still work?

Reply:
No, that would cause this to break. Good catch. We should be able to use the named shape.